lib/livebook/teams/connection.ex

```diff
@@ -87,6 +87,14 @@ defmodule Livebook.Teams.Connection do
     handle_websocket_message(message, data)
   end
 
+  def handle_event(:info, {:ssl_closed, _port} = message, @no_state, %__MODULE__{} = data) do
+    handle_websocket_message(message, data)
+  end
```

Comment:
Not sure if this is expected

Reply:
There are a couple more listed [here](https://www.erlang.org/doc/man/ssl.html#type-active_msgs). If we want Mint to handle all of them, perhaps we should just do `when elem(message, 0) in [...]` with all the tags?

Reply:
From a quick look it looks like they handle it, so let's do that :)

Reply:
FTR [here](https://github.com/elixir-mint/mint_web_socket/blob/f44a35ef7883008d435091c96e06a41d60849601/lib/mint/web_socket.ex#L401-L407) and [here](https://github.com/elixir-mint/mint/blob/371f4d7bffc26a779ac7675ae75f89d17519bfa9/lib/mint/http.ex#L140-L147).

Reply:
Done